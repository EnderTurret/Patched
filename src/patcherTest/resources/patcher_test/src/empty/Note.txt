This directory must be empty at all times!