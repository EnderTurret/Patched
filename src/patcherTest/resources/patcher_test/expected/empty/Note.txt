This folder is now reserved for memes